### PR TITLE
Changes/fixes to almost every line of copy in the game.

### PR DESCRIPTION
**Pull request by neue**
- Made colouring consistent 
- Added colouring to every object/character referenced in the game that is also an object in the file system
- Added quotation marks to speech in the 2nd half of the game
- ASCII art scrolls for commands

### CODE REVIEW

File: linux_story/ascii_assets/story_files/LS [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+|  This includes your current position . and the                |
+|  containing directory ..                                      |
+|                                                               |
+|  Example                                                      |
+|                                                               |
+|  ls -a ../                                                    |
+|  Will show you everything in the directory behind you         |____
+|                                                               |__( \
+\                                                               \    /
+ `---------------------------------------------------------------`=='
```
Comment (radujipa) [inline linux_story/ascii_assets/story_files/LS]: Does this scroll fit on the screen when you `cat` it?

Comment (neue) [inline linux_story/ascii_assets/story_files/LS]: It does now

Comment (radujipa) [inline linux_story/ascii_assets/story_files/LS]: Could you maybe move the `ls -a` to under `ls` and explain that it should the hidden items?

**Comment by radujipa** [comment]
Nice work mate :shipit:

**Pull request merged**